i6_models/parts/fsa.py

```diff
@@ -0,0 +1,136 @@
+__all__ = ["TorchFsaBuilder", "WeightedFsa"]
+
+from functools import reduce
+from typing import Iterable, NamedTuple, Tuple, TypeVar
+
+import numpy as np
+import torch
+
+TWeightedFsa = TypeVar("TWeightedFsa", bound="WeightedFsa")
+
+
+class WeightedFsa(NamedTuple):
+    """
+    Convenience class that represents an FSA. It supports scaling the weights of the
+    fsa by simple left-multiplication and moving the tensors to a different device.
+    It can simply be passed to `i6_native_ops.fbw.fbw_loss` and `i6_native_ops.fast_viterbi.align_viterbi`.
+    :param num_states: the total number of all states S
```

Comment:
```suggestion
    It can simply be passed to `i6_native_ops.fbw.fbw_loss` and `i6_native_ops.fast_viterbi.align_viterbi`.

    :param num_states: the total number of all states S
```
There should be an empty line between the description and the params in the docstring.